basic/line.go

```diff
@@ -14,7 +14,7 @@ func (Line) String() string { return "Line" }
 // NewLine creates a line given pairs for floats.
 func NewLine(pointPairs ...float64) *Line {
 	line := new(Line)
-	if (2 % len(pointPairs)) != 0 {
+	if (len(pointPairs) % 2) != 0 {
 		panic("NewLine requires pair of points.")
```

Comment:
Do we want to set a precedence that a panic is ok in this situation? I understand we avoid returning an error which is nice...